pysubtools/parsers/microdvd.py

```diff
@@ -101,12 +101,12 @@ def _parse_header(self, header, global_only=False):
                 t['styles']['*']['font-family'] = v.strip()
             elif k.lower() == 's':
                 # Font size
-                t['styles']['*']['font-size'] = v.strip() + \
-                    ('px' if v.strip().isdigit() else '')
+                t['styles']['*']['font-size'] = v.strip()
+                + ('px' if v.strip().isdigit() else '')
```

Comment:
Are you sure this code works? I think you need `\` or wrap whole expression into `()`.